### ISSUE TITLE
Incorrect GitHub diff URL for deps with `:git/url` that ends in `.git`

### ISSUE THREAD

**Issue by p-himik** [liquidz/antq#61]
My `deps.end` has this dependency in it:
```
{juxt/kick.alpha {:git/url "https://github.com/juxt/kick.alpha.git"
                  :sha     "e498d06fb99a82c808c2436574fae643016abbc0"}}
```
An abridged output of antq:
```
|    :file |                           :name |                                 :current |                                  :latest |
|          |                 juxt/kick.alpha | e498d06fb99a82c808c2436574fae643016abbc0 | e552e314750b4c5f337fe44849d92725877dffb0 |

Available diffs:
- https://github.com/juxt/kick.alpha.git/compare/e498d06fb99a82c808c2436574fae643016abbc0...e552e314750b4c5f337fe44849d92725877dffb0
```

Notice how the URL includes `kick.alpha.git` whereas it should be just `kick.alpha`.

**Comment by liquidz** [comment]
@p-himik Thank you for reporting!
Oh I missed that GitHub adds ".git" to the end of default clone URL.

**Comment by p-himik** [comment]
Yeah, but note that it's optional - one can remove `.git` and it will work just fine.
I wonder if you could create projects that have their name ending in `.git`. Then there's a possibility for a URL that ends in `.git.git` and a solution that just removes that `.git` part won't work.

**Comment by liquidz** [comment]
It may be no problem.
![image](https://user-images.githubusercontent.com/6941/109402409-01acef80-7999-11eb-917b-456a359c064c.png)

**Comment by liquidz** [comment]
@p-himik I just released v0.11.2 ☺️  #62

**Comment by p-himik** [comment]
Awesome, thanks!
